doc/source/release-notes/Percona-Server-5.6.33-79.0.rst

```diff
@@ -0,0 +1,118 @@
+.. rn:: 5.6.33-79.0
+
+============================
+|Percona Server| 5.6.33-79.0
+============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.33-79.0 on
+October 18th, 2016 (Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.33-79.0/>`_
+and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.6.33
+<http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-33.html>`_, including
+all the bug fixes in it, |Percona Server| 5.6.33-79.0 is the current GA release
+in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source
+and free, all the details of the release can be found in the `5.6.33-79.0
+milestone at Launchpad
+<https://launchpad.net/percona-server/+milestone/5.6.33-79.0>`_.
+
+New Features
+============
+
+ |Percona Server| has implemented support for :ref:`compressed_columns`.
+
+ A new |TokuDB| :variable:`tokudb_dir_per_db` option has been introduced to
+ address two |TokuDB| shortcomings, the :ref:`renaming of data files
+ <improved_table_renaming_functionality>` on table/index rename, and the
+ ability to :ref:`group data files together
+ <improved_directory_layout_functionality>` within a directory that represents
+ a single database. This feature is disabled by default.
+
+Bugs Fixed
+==========
+
+ After fixing bug :bug:`1540338`, system table engine validation check is no
+ longer skipped for tables that don't have an explicit ``ENGINE`` clause in
+ a ``CREATE TABLE`` statement. If |MySQL| upgrade statements are replicated,
+ and slave does not have the |MyISAM| set as a default storage engine, then
+ the ``CREATE TABLE mysql.server`` statement would attempt to create an
+ |InnoDB| table and fail because ``mysql_system_tables.sql`` script omitted
+ explicit engine setting for this table. Bug fixed :bug:`1600056`.
+ 
+ :ref:`audit_log_plugin` malformed record could be written after
+ :variable:`audit_log_flush` was set to ``ON`` in ``ASYNC`` and ``PERFORMANCE``
+ modes. Bug fixed :bug:`1613650`.
+
+ A race condition between :ref:`handlersocket_page` and server shutdown could
+ lead to a server stall if shutdown was issued immediately after
+ :ref:`handlersocket_page` startup. Bug fixed :bug:`1617198`.
+
+ :ref:`handlersocket_page` could access freed memory on startup. Bug fixed
+ :bug:`1617998`.
+
+ Running ``LOCK TABLES`` with :ref:`threadpool` could lead to a deadlock. Bug
```

Comment:
Workloads with statements that take non-transactional locks (LOCK TABLES, global read lock, and similar) could have caused deadlocks when running under threadpool with high priority queue enabled and thread_pool_high_prio_mode set to transactions. Fixed by placing such statements into the high priority queue even with the above thread_pool_high_prio_mode setting